src/class/gc.rs

```diff
@@ -110,7 +110,7 @@ impl GC {
     ///
     /// # Examples
     ///
-    /// ```
+    /// ```no_run
```

Comment:
@goyox86 Check a better way of running these ones only in the right Ruby versions? Worth it? ruby2.7.* is phased out.

Reply:
✅